Havit.Blazor.Components.Web.Bootstrap/wwwroot/HxAutosuggest.js

```diff
@@ -1,4 +1,18 @@
-﻿export function open(inputElement, hxAutosuggestDotnetObjectReference) {
+﻿export function initialize(inputId, hxAutosuggestDotnetObjectReference, keys) {
+    let inputElement = document.getElementById(inputId);
+
+    inputElement.onkeydown = function (e) {
+        let key = e.key;
+
+        hxAutosuggestDotnetObjectReference.invokeMethodAsync("HxAutosuggest_HandleInputKeyDown", key);
+
+        if (keys.includes(key)) {
+            e.preventDefault();
+        }
+    }
```

Comment:
Should we stick with the pattern where the event handler is a standalone method and we unsubscribe from the event in dispose?